index.html

```diff
@@ -784,10 +784,18 @@ <h3>Proofs</h3>
 
           <dt>verificationMethod</dt>
           <dd>
-The means and information needed to verify the proof MUST be specified as a
-string that maps to a [[URL]]. An example of a verification method is a
-link to a [=public key=] which includes cryptographic material that is used
-by a verifier during the verification process.
+If included, `verificationMethod`, the means and information needed to verify the proof,
+MUST be specified as a string that maps to a [[URL]]. An example of a verification method
+is a link to a [=public key=] which includes cryptographic material that is used
+by a verifier during the verification process. Inclusion of `verificationMethod` is OPTIONAL, 
+but if it is not included, other properties such as `cryptosuite` MUST provide a mechanism
+by which to obtain the required information to verify the proof.
+          </dd>
+
+          <dt>cryptosuite</dt>
+          <dd>
+Information about the cryptographic suite that was used to generate the proof. See
+[[[#cryptographic-suites]]] for more information. `cryptosuite` is REQUIRED.
```

Comment:
```suggestion
An identifier for the cryptographic suite that can be used to verify the proof. See
[[[#cryptographic-suites]]] for more information. `cryptosuite` MUST be specified
and its value MUST be a string.
```

Reply:
Hmm, on second glance, this is wrong. `cryptosuite` is not a requirement for general Proof objects, they are a requirement for `DataIntegrityProofs` which is a specific type of Proof object. This requirement would break all legacy implementations.

Reply:
Yeah, missed this, `cryptosuite` is only required when the type is `DataIntegrityProof`.

Reply:
Does this work, @msporny, @dlongley?
```suggestion
An identifier for the cryptographic suite that can be used to verify the proof. See
[[[#cryptographic-suites]]] for more information. If the proof type is
`DataIntegrityProof`, `cryptosuite` MUST be specified; otherwise,  `cryptosuite`
MAY be specified. If specified, its value MUST be a string.
```